homeassistant/components/zwave_js/climate.py

```diff
@@ -336,9 +336,11 @@ def device_state_attributes(self) -> Optional[Dict[str, str]]:
     def supported_features(self) -> int:
         """Return the list of supported features."""
         support = SUPPORT_PRESET_MODE
-        if len(self._current_mode_setpoint_enums) == 1:
+        # If any setpoint value exists, we can assume temperature
+        # can be set
+        if any(self._setpoint_values.values()):
```

Comment:
Do we need `any`?

Reply:
we set `self._setpoint_values[enum]` to an `Optional[ZwaveValue]`. The `any` here is to check if any value was found, which means that the thermostat has a setpoint